python-malformed-yaml/Dockerfile

```
@@ -0,0 +1,13 @@
+FROM python:3.12-slim
+
+RUN apt-get update
```

```suggestion
RUN apt-get update && apt-get upgrade -y && apt-get clean && rm -rf /var/lib/apt/lists/*
```
I think we need upgrade in here as well to apply the updates. Running this is the same `RUN` statement is good for the image size.